### ISSUE TITLE
Strange upgrade phenomenon, I don't know if this is normal.

### ISSUE THREAD

**Issue by smartluo1833** [IllusionMods/KKManager#30]
**Updating automatically deletes GB update data previously downloaded?
Second Checked Update show 0 B, why I need to download 7.69GB + 2.24GB again when Third Checked Update??**

KKManager 0.9.3
[KKManager.log](https://github.com/IllusionMods/KKManager/files/4883533/KKManager.log)

1. First Checked Upgrade
Before: Koikatu+HF Patch 3.2 ≈ 26GB ↓
After: Koikatu+HF Patch 3.2 + First Upgrade ≈ 35.6GB
![3](https://user-images.githubusercontent.com/60388814/86745520-fd7c3c80-c06c-11ea-9845-ca31b40954a1.jpg)

2. Second Checked Upgrade
Before: Koikatu+HF Patch 3.2 + First Upgrade ≈ 35.6GB ↓
After: Koikatu+HF Patch 3.2 + Second Upgrade ≈  29.2GB
![4](https://user-images.githubusercontent.com/60388814/86745551-03721d80-c06d-11ea-8d59-428bc33200b7.jpg)

3. Third Checked Upgrade
Before: Koikatu+HF Patch 3.2 + Second Upgrade ≈  29.2GB   
Second Checked Update show 0 B, why I need to download 7.69GB + 2.24GB again??
![5](https://user-images.githubusercontent.com/60388814/86745580-0836d180-c06d-11ea-99c0-23031f3f283e.jpg)

**Comment by ManlyMarco** [comment]
Your KKManager version is way too old, update to the latest version.

**Comment by smartluo1833** [comment]
@ManlyMarco 

Thank you ManlyMarco.
Should I wait for the next version of KKManager?

[KKManager.log](https://github.com/IllusionMods/KKManager/files/4886184/KKManager.log)
I did what you said, Koikatu →installed HF Patch 3.2 ✔all→ KKManager 0.12.0
Then I stopped at “Look for updates”    searching and searching.........
I turned off the firewall.
I tried US, Japan, Singapore paid VPN and unfortunately got the same result "searching and searching........."
I cmd ping ai-shoujo.com -t
sometimes  lose 0%, sometimes  lose 16%. 
0.9.3 can work but have problem  
as you said 
0.10 Supports new version of mod update info format (old KKManager versions won't be able to update using these)
I think I should not use 0.9.3
